hub/package-manager/winget/pinning.md

```diff
@@ -0,0 +1,234 @@
+---
+title: The winget pin command
+description: Use the winget pin command and subcommands to list and manage the repositories Windows Package Manager accesses.
+ms.date: 06/16/2023
+ms.topic: reference
+ms.localizationpriority: medium
+ms.custom: kr2b-contr-experiment
+---
+
+# The winget pin command
+
+The [winget](index.md) tool **pin** command allows you to limit the Windows Package Manager from upgrading a package to specific ranges of versions, or it can prevent it from upgrading a package altogether. A pinned package may still upgrade on its own and be upgraded from outside the Windows Package Manager. 
+
+WinGet pinning supports three types of Package Pinning:
+- **Pinning**:
+The package is excluded from winget upgrade --all but allows winget upgrade <WinGet package>. You can use the "–include-pinned” argument to let winget upgrade --all include pinned packages.
```

Comment:
> `<WinGet package>`

This seems to be messing with the rendering of the markdown file since it looks like an HTML tag. Not sure if on the actual site it would look different. Maybe we could use a different type of markers for placeholders.